app/scripts/modules/core/src/pipeline/config/stages/travis/travisStage.ts

```diff
@@ -126,15 +133,21 @@ export class TravisStage implements IController {
     }
   }
 
-  public updateParam(parameter: any): void {
-    if (this.useDefaultParameters[parameter] === true) {
-      delete this.userSuppliedParameters[parameter];
-      delete this.stage.parameters[parameter];
-    } else if (this.userSuppliedParameters[parameter]) {
-      this.stage.parameters[parameter] = this.userSuppliedParameters[parameter];
-    }
+  public addParameter(): void {
+    this.$uibModal.open({
+      templateUrl: require('./modal/addParameter.html'),
+      controller: 'TravisStageAddParameterCtrl',
+      controllerAs: 'addParameter',
```

Comment:
better to use `ctrl` (for non-components) or `$ctrl` (for components) - it's a little more conventional and avoids confusion about where things are defined for others when they are looking at the template